### PR TITLE
School Districts

### PR DESCRIPTION
**Pull request by miriamseely**
Current draft of the [Utah School District Boundaries](https://gis.utah.gov/products/sgid/boundaries/school-districts/) layer from the Boundaries index (not the Utah School Board Districts from the Political index).

### CODE REVIEW

File: metadata/boundaries/school_district_boundaries.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Utah School District Boundaries
+
+## ID
+
+fb17c1bb-954a-4a3b-997a-7024cae04fbc
+
+## Brief Summary
+
+Polygon dataset representing more than 40 school districts in Utah.
```
Comment (steveoh) [inline metadata/boundaries/school_district_boundaries.md:11]: **question**: should we add what types of schools since we have a few different layers?

**Review by steveoh — COMMENTED** [review]
looks pretty normal

File: metadata/boundaries/school_district_boundaries.md (hunk 1 above)
Comment (jacobdadams) [inline metadata/boundaries/school_district_boundaries.md:11]:
```suggestion
Polygon dataset representing the more than 40 school districts in Utah.
```
"The" implies it is an exhaustive dataset.

File: metadata/boundaries/school_district_boundaries.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Summary
+
+This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. We maintain this layer using data sent in by School District Offices and certified through the Lieutenant Governor's Office.
```
Comment (jacobdadams) [inline metadata/boundaries/school_district_boundaries.md:15]:
```suggestion
This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. We maintain this layer using data provided by the school district offices and certified through the Lieutenant Governor's Office.
```
I like repetition/parallelism :)

File: metadata/boundaries/school_district_boundaries.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## Description
+
+### What is the dataset?
+
+This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. School districts are independently operated government units managed by an elected school board. The majority of school districts in Utah are county-wide, but there are a few that are smaller than the county they reside in (e.g. Alpine, Granite, and Jordan School Districts).
```
Comment (jacobdadams) [inline metadata/boundaries/school_district_boundaries.md:21]:
```suggestion
This dataset is comprised of polygons representing the approximate geographic extent of all school districts in Utah. School districts are independently-operated government units managed by a publicly-elected school board. The majority of school districts in Utah are county-wide, but a few are smaller than the county they reside in (e.g. Alpine, Granite, and Jordan School Districts).
```

File: metadata/boundaries/school_district_boundaries.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+### What does the dataset represent?
+
+School district boundaries are represented as polygons and include the school district name and total enrollment according to the Utah State Board of Education 2015-2016 [Fingertip Facts](https://schools.utah.gov/fingertipfacts) page.
+
+<!--- The latest Fingertip Facts sheet is from 2019-2020. Are we planning on updating the total enrollment as the fingertip facts come out? -->
+
+### How was the dataset created?
+
+We developed this dataset using coordinate geometry ([COGO](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm)) from legal descriptions provided by the Utah State Tax Commission and the Utah State Board of Education. UGRC maintains this dataset by aggregating new data received by school district offices. All updates, corrections, or clarifications to the school district boundaries represented in this dataset must first be filed with the [Lieutenant Governor's](https://ltgovernor.utah.gov/) office. Once approved, UGRC adjusts the polygon boundaries accordingly.
```
Comment (jacobdadams) [inline metadata/boundaries/school_district_boundaries.md:35]:
```suggestion
UGRC used coordinate geometry ([COGO](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm)) from legal descriptions provided by the Utah State Tax Commission and the Utah State Board of Education to create this dataset. We maintain it by aggregating new data received from school district offices. All updates, corrections, or clarifications to the school district boundaries represented in this dataset must first be filed with the [Lieutenant Governor's](https://ltgovernor.utah.gov/) office. Once approved, UGRC adjusts the polygon boundaries accordingly.
```

**Review by jacobdadams — APPROVED** [review]
I like this one too. Just a few grammatical changes, cause I'm weird like that.

File: metadata/boundaries/school_district_boundaries.md (hunk 1 above)
Comment (miriamseely) [reply metadata/boundaries/school_district_boundaries.md]: I feel adding the "the" makes it sound awkward. I changed it to "Polygon dataset representing geographic boundaries of all school districts in Utah." Thoughts?

**Pull request merged**